base/Base.jl

```diff
@@ -232,6 +232,9 @@ using .Libc: getpid, gethostname, time
 
 include("env.jl")
 
+# OpaqueClosure
+include("opaque_closure.jl")
```

Comment:
I suspect this should be in `Experimental`, for now?

Reply:
Sure